build/credscan-exclusion.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Merging the ADAL.NET and MSAL.NET credscan's exclusion file for the common build.